sites/next.skeleton.dev/src/examples/components/rating/AllowHalf.svelte

```diff
@@ -0,0 +1,5 @@
+<script lang="ts">
```

Comment:
Ratings examples do not follow the file naming convention. Should always be:

```
Example + {Description} + {extension}
```

Such as:

* Example.svelte | Example.tsx (the default)
* ExampleFoo.svelte | ExampleFoo.tsx
* ExampleBar.svelte | ExampleBar.tsx

This helps ensure don't have overlaps with custom components generated for the Svelte app. I know the path will differ, but this helps keep things explicit.

Reply:
Done.